core/templates/dev/head/components/forms/schema_viewers/SchemaBasedDictViewerDirective.js

```diff
@@ -31,7 +31,7 @@ oppia.directive('schemaBasedDictViewer', [
       },
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/components/forms/schema_viewers/' +
-        'schema_based_dict_viewer.html'),
```

Comment:
@seanlip: No such file exists. I think it refers to the corresponding directive. Therefore changed accordingly. Thanks!

Reply:
Great find! Do you know why it worked before?

Reply:
Oh, maybe it's because we just don't use the viewer directive for dict.

Reply:
Yes, that might be the case!